recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "6.1.0" %}
+{% set version = "6.2.0" %}
```

Comment:
There should be the latest version `6.3.0`

Reply:
the package was updated to the lates version `6.3.0`:
https://github.com/AnacondaRecipes/nbconvert-feedstock/blob/update_to_6.2.0/recipe/meta.yaml#L1